src/assets/css/style.css

```diff
@@ -1,10 +1,140 @@
-body {
-  margin: 0;
-}
-
-h1 {
-  font-size: 30px;
-  font-family: -apple-system, blinkmacsystemfont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif;
-}
-
-/*# sourceMappingURL=style.css.map */
+body {
+  margin: 0;
+  background-color: #fafafa !important;
+}
+
+input {
+  border: 2px solid #f4f5f9;
+  outline: none;
+}
+
+h1 {
+  font-size: 30px;
+  font-family: -apple-system, blinkmacsystemfont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif;
+}
+
+.listItems {
+  max-height: 370px;
+  overflow-y: scroll;
+}
+
+.listItems::-webkit-scrollbar {
+  width: 5px;
+}
+
+.listItems::-webkit-scrollbar-track {
+  -webkit-box-shadow: inset 0 0 6px #f4f5f9;
+  border-radius: 10px;
+}
+
+.listItems::-webkit-scrollbar-thumb {
+  border-radius: 5px;
+  background: #0290ff;
+}
+
+.header {
+  height: 70px;
+  background-color: white;
+  border-bottom: 3px solid #f4f5f9;
+}
+.header .container {
+  height: 100%;
+}
+.header .container .left h6 {
+  color: #0290ff;
+  font-size: 20px;
+  font-family: -apple-system, blinkmacsystemfont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif;
+  font-weight: 900;
+}
+.header .container .right span {
+  width: 40px;
+  height: 40px;
+  border-radius: 50%;
+  border: 2px solid #f4f5f9;
```

Comment:
`.header .container .left h6 {...} ` and `.header .container .right span` both have a couple of similar style rules. Group them together.